### PR TITLE
implement initpin function in p11mod #128

### PR DESCRIPTION
**Pull request by KingAnointing**
I had alot of bug unidentified bug in the last branch i had to delete it

### CODE REVIEW

File: p11mod/p11mod.go [hunk 1]
```diff
@@ -222,9 +222,20 @@ func (ll *llBackend) GetMechanismInfo(slotID uint, m []*pkcs11.Mechanism) (pkcs1
...
-	// TODO
-	log.Println("p11mod InitPIN: not implemented")
-	return pkcs11.Error(pkcs11.CKR_FUNCTION_NOT_SUPPORTED)
+	session, err := ll.getSessionByHandle(sh)
+	if err != nil {
+		return err
+	}
+
+	if trace {
+		log.Printf("p11mod initpin")
```
Comment (JeremyRand) [inline p11mod/p11mod.go:231]: Please capitalize "InitPIN" to match the function name.

**Comment by JeremyRand** [comment]
ACK 4a52a2d58708e679d8dbea482390ef0042ccd001, subject to figuring out what the deal is with the Cirrus compile fails.

**Comment by JeremyRand** [comment]
I think the Cirrus fails are spurious, I just triggered a rebuild.

**Comment by JeremyRand** [comment]
ACK 4a52a2d58708e679d8dbea482390ef0042ccd001

File: p11mod/p11mod.go (hunk 1 above)
Comment (KingAnointing) [reply p11mod/p11mod.go]: Done ✅

**Pull request merged**